### PR TITLE
Add support for direction tag into viewpoint pois

### PR DESCRIPTION
**Pull request by rwrx**
Add support for direction tag into viewpoint pois.

Connects with https://github.com/tilezen/vector-datasource/issues/598.

### CODE REVIEW

**Comment by nvkelso** [comment]
👀

**Comment by nvkelso** [comment]
This connects with https://github.com/tilezen/vector-datasource/issues/598 in the v1.9.0 milestone.

File: yaml/pois.yaml [hunk 1]
```diff
@@ -2206,6 +2206,7 @@ filters:
     output:
       <<: *output_properties
       kind: viewpoint
+      direction: {col: direction}
```
Comment (nvkelso) [inline yaml/pois.yaml]: There are many more POIs that include `direction`, so let's move this to the very top of the YAML doc after     `wikidata_id: {col: wikidata}` in the 
 

```
global:
  - &output_properties:
```

section, please.

**Comment by rwrx** [comment]
@nvkelso I am sorry for this late answer, I was quite busy and it is also summer time :). I have tried to implement it as you wrote, but I am not sure if I understand it correctly, so if needed I can make further changes. Also could I ask you for advise how to write tests for this?

**Comment by nvkelso** [comment]
This looks good :) I'll have a look this week and add the tests, thank you!

**Comment by nvkelso** [comment]
Going to merge this and add tests in new PR.

**Comment by nvkelso** [comment]
Fixed a few edge case bugs and added tests, docs in https://github.com/tilezen/vector-datasource/pull/1921.
